.travis.yml

```diff
@@ -0,0 +1,62 @@
+language: generic
+sudo: required
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+
+
+# Cache shennanigans
```

Comment:
I left this commented stuff in here, because it _may_ be worth having in the future.
Currently, with the linting and testing taking next to no time, it's more overhead than it's worth, but later on when testing takes longer, it may then be worth dumping the docker images in cache.